UpdateProfile/handler.ts

```diff
@@ -103,11 +103,27 @@ export function UpdateProfileHandler(
       emailChanged ? false : existingProfile.isEmailValidated
     );
 
+    const autoEnableInboxAndWebHook =
+      existingProfile.acceptedTosVersion === undefined &&
+      profile.acceptedTosVersion !== undefined;
+
     const errorOrMaybeUpdatedProfile = await profileModel
       .update({
         ...existingProfile,
         // Remove undefined values to avoid overriding already existing profile properties
-        ...withoutUndefinedValues(profile)
+        ...withoutUndefinedValues(profile),
+
+        // Bugfix missing enable of Inbox and Webhook https://www.pivotaltracker.com/story/show/175095963
```

Comment:
This is not a proper bugfix.
We are shifting this logic from the app to the backend.

Reply:
Removed this comment, the link was added to another comment
4f6d4ea